src/tests/Interop/IJW/CopyConstructorMarshaler/CopyConstructorMarshaler.cs

```diff
@@ -62,6 +62,7 @@ public static int TestEntryPoint()
         }
 
         [Fact]
+        [SkipOnCoreClr("JitStress and MinOpts can introduce extra copies that won't happen in production scenarios.", RuntimeTestModes.JitStress | RuntimeTestModes.JitMinOpts)]
```

Comment:
MinOpts is a production scenario.

Reply:
I'll test this again with MinOpts and make sure it works.